### PR TITLE
communities: community's logo

### PR DESCRIPTION
**Pull request by bouzlibop**

### CODE REVIEW

**Comment by bouzlibop** [comment]
They were integrated into `zenodo/invenio` at some point, but (I think) never integrated in `invenio/invenio` or later `invenio/invenio-communities`.

- https://github.com/zenodo/invenio/commit/b602adef4ea7dbb938ac7e13e89a3015b0b6d7f1
- https://github.com/zenodo/invenio/commit/8e368c312fb3aa308056911d70da414f4c986f31

File: invenio_communities/templates/communities/new_base.html [hunk 1]
```diff
@@ -139,6 +139,12 @@ <h3 id="deletemodal_label">Delete community?</h3>
                                     {{ field.label.text }}
                                 </label>
                                 <div class="controls col-md-9">
+                                    {%- if field.type == "FileField" and community.logo_url %}
+                                      <div>
+                                        <img src="{{ community.logo_url }}" style="display: block; margin: auto; max-width: 100%; max-height: 200px;">
```
Comment (jirikuncar) [inline invenio_communities/templates/communities/new_base.html]: can you migrate the `style` attribute to `class` and `css`?

File: invenio_communities/views.py [hunk 2]
```diff
@@ -282,12 +283,28 @@ def new():
...
         del data['identifier']
-        c = Community(id_user=uid, **data)
-        db.session.add(c)
-        db.session.commit()
-        c.save_collections()
-        flash("Community was successfully created.", category='success')
-        return redirect(url_for('.index'))
+        del data['logo']
+        logo_ext = None
+        file = request.files.get('logo', None)
```
Comment (jirikuncar) [inline invenio_communities/views.py]: please choose a different variable name - `file` is Python building function

**Comment by jirikuncar** [comment]
closed via #19 (d9bec263e753b1bcbc4d0ea89982159729e44595).

**Pull request closed without merging**